src/Microsoft.Extensions.Logging/LoggingServiceCollectionExtensions.cs

```diff
@@ -19,14 +20,31 @@ public static class LoggingServiceCollectionExtensions
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
         public static IServiceCollection AddLogging(this IServiceCollection services)
         {
+            return AddLogging(services, builder => { });
+        }
+
+        /// <summary>
+        /// Adds logging services to the specified <see cref="IServiceCollection" />.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configure">The <see cref="ILoggerBuilder"/> configuration delegate.</param>
+        /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
+        public static IServiceCollection AddLogging(this IServiceCollection services, Action<ILoggerBuilder> configure)
```

Comment:
Nitpick: Wouldn't `ILoggingBuilder` be better since the "Logging" module is configured with it and not a single logger. (just a suggestion/question, probably really not important)